modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,23 @@
+= B2B Solutions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: b2b, edi, portal
+
+== B2B (Business-to-Business)
+
```

Comment:
Why are you defining b2b?

Reply:
Amit, David, and feel there is value in talking about B2B here as it relates to Anypoint Partner Manager. It may be defined in the Global Glossary, but feel it is okay for making it a building block to what Anypoint Partner Manager does within B2B.